src/js/events/AppVersionsEvents.js

```diff
@@ -0,0 +1,9 @@
+var AppVersionsEvents = {
+  CHANGE: "APP_VERSIONS_EVENTS_CHANGE",
+  REQUEST_APP_VERSIONS: "APP_VERSIONS_EVENTS_REQUEST_APP_VERSIONS",
+  REQUEST_APP_VERSIONS_ERROR: "APP_VERSIONS_EVENTS_REQUEST_APP_VERSIONS_ERROR",
+  REQUEST_APP_VERSION: "APP_VERSIONS_EVENTS_REQUEST_APP_VERSION",
+  REQUEST_APP_VERSION_ERROR: "APP_VERSIONS_EVENTS_REQUEST_APP_VERSION_ERROR"
```

Comment:
Oh yes, that is much better.
I changed the one to 
```
REQUEST_VERSION_TIMESTAMPS
```
because it turns out that is the actually meaning of this request.